packages/extension-chakra-storefront/jest.config.js

```diff
@@ -125,10 +62,10 @@ module.exports = {
     // TODO: Revert this threshold once we start adding tests back to the codebase
     coverageThreshold: {
         global: {
-            // statements: 73,
-            // branches: 60,
-            // functions: 65,
-            // lines: 74
+            statements: 73,
+            branches: 60,
+            functions: 65,
+            lines: 74
```

Comment:
We should start bumping these number up when we get to address all the broken tests